allure-generator/src/main/javascript/components/side-nav/styles.scss

```diff
@@ -44,7 +44,7 @@
 
   &__footer {
     margin-top: auto;
-    margin-bottom: $gap-size;
+    margin-bottom: $footer-gap-size;
```

Comment:
I have added another gap-size const variable in order to lift the languages bar a little upper.
When I added Hebrew and opened the language bar, the Hebrew language was sliced by the page (overflow).